proofOfConcept/ColorPickerAlpha/ColorPickerAlpha/MainWindow.xaml

```diff
@@ -5,7 +5,9 @@
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:local="clr-namespace:ColorPickerAlpha"
         mc:Ignorable="d"
-        Title="MainWindow" Height="78" Width="599" Background="#FF4472C4">
+        Title="MainWindow" Height="60" Width="600" Background="#FF4472C4"
```

Comment:
Is there a reason height and width are hard coded?  Hard coding dialog can be a huge problem when resolutions change and cause you all sorts of confusion issues.  Just a thought, not a show stopper.  Try dragging the window from your external (1080P) monitor to your SurfaceBook monitor.  What happens?  If all is good, then probably ok.